lm-agent/tests/parsing/test_flexlm.py

```diff
@@ -53,31 +53,31 @@ def test_parse_feature_line(line, result):
     "line,result",
     [
         (
-            "    user1 myserver.example.com /dev/tty (v62.2) (myserver.example.com/24200 12507), start Thu 10/29 8:09, 29 licenses",
+            "    user1 myserver.example.com /dev/tty (v62.2) (myserver.example.com/24200 12507), start Thu 10/29 8:09, 29 licenses",  # noqa: E501
```

Comment:
[optional] Configuring a per-file ignore can also work in this case.

Reply:
Good idea, the files that have more than one E501 are excluded now.